sdk/core/azure-core/src/main/java/module-info.java

```diff
@@ -36,6 +36,7 @@
     opens com.azure.core.implementation.logging to com.fasterxml.jackson.databind;
     opens com.azure.core.implementation.serializer to com.fasterxml.jackson.databind;
     opens com.azure.core.implementation.serializer.jsonwrapper to com.fasterxml.jackson.databind;
+    exports com.azure.core.util.paging;
```

Comment:
Keep all `exports` together (move this to line 29)

Reply:
Changed ordering